hadoop-spark/steps/01_logging/after-deploy

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eux
+
+. "$CONJURE_UP_SPELLSDIR/sdk/common.sh"
+
+if [[ "$LOGGINGPLUGIN" == "Rsyslog" ]]; then
+    ip=$(unitAddress rsyslog)
```

Comment:
I think this needs to be `$(unitAddress rsyslog 0)` or whatever the unit is we want to query for address